src/test/kotlin/accountTest/GavetaTest.kt

```diff
@@ -0,0 +1,36 @@
+package accountTest
+
+import account.Gaveta
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+
+class GavetaTest {
+
+    @Test
```

Comment:
Uma dica: quando for realizar testes para um mesmo cenário com expectativas diferentes, [existe uma forma de agrupar contextos que são comuns](https://kotlintesting.com/nested-tests-in-junit5/). Por exemplo, esses 2 primeiros testes referem-se ao `abreGavetaDeposito`.

Uma forma possível de organizar seria:

```kotlin
@Nested
inner class `#abreGavetaDeposito` {
  @Test
  fun `ira retornar mensagem de valores conferidos caso valores sejam iguais`() {
     ...
  }

  @Test
  fun `ira retornar uma exception em caso de valor inserido diferente do informado`() {
     ...
  }
}
```

Reply:
(Não precisa alterar, fica só como dica msm)